### PR TITLE
Post-deploy client network configuration

### PR DESCRIPTION
**Pull request by jaywcarman**

### CODE REVIEW

File: scripts/python/lib/config.py [hunk 1]
```diff
@@ -1504,6 +1526,67 @@ def yield_ntmpl_roles(self, node_template_index):
         for member in self.get_ntmpl_roles(node_template_index):
             yield member
 
+    def get_ntmpl_interfaces(self, node_template_index):
+        """Get node_templates interfaces
+        Args:
+            node_template_index (int): Node template index
+
+        Returns:
+            list of str: List of interface labels
```
Comment (rayjh) [inline scripts/python/lib/config.py]: This says it's returning a list of interface labels, but the code looks to be returning more than that.

File: scripts/python/lib/config.py [hunk 2]
```diff
@@ -1871,19 +2000,58 @@ def yield_ntmpl_phyintf_data_switch(self, node_template_index):
...
+        Returns:
+            str: PXE interface device (iface) value
+        """
+
+        node_template = self.cfg.node_templates[node_template_index]
+        if_label = node_template.physical_interfaces.pxe[index].interface
+        return self.lookup_interface_iface(if_label)
+
+    def lookup_interface_iface(self, if_label):
+        """Get node_templates physical_interfaces data dev
```
Comment (rayjh) [inline scripts/python/lib/config.py]: 'Get node_templates physical_interfaces data dev' should probably say get interface template data device'

File: scripts/python/lib/config.py (hunk 1 above)
Comment (jaywcarman) [reply scripts/python/lib/config.py]: Good catch, it is returning an interface dictionary.

File: scripts/python/lib/config.py (hunk 2 above)
Comment (jaywcarman) [reply scripts/python/lib/config.py]: Agreed.

**Pull request merged**